stack-ghc7.10.yaml

```diff
@@ -26,8 +26,8 @@ extra-deps:
 - base-orphans-0.7
 - bifunctors-5.5.2
 - brick-0.37.1
-- cassava-megaparsec-1.0.0
-- config-ini-0.2.2.0
+- cassava-megaparsec-2.0.0
```

Comment:
https://www.stackage.org/package/cassava-megaparsec hasn't been in stackage for a long time, which complicates our install instructions. We'll try and improve that.